### ISSUE TITLE
About the GPU usage in the Pytorch code

### ISSUE THREAD

**Issue by vinthony** [mahmoudnafifi/CIE_XYZ_NET#2]
Hi, Afifi,

Awesome work!

I have tried your code for a larger image. However, I find something that might cause unnecessary memory usage in your PyTorch evaluation code. 
The core torch model should be wrapped in 
```
with torch.no_grad():
```
to avoid the calculation of gradient when testing.

More details can be found here: https://datascience.stackexchange.com/questions/32651/what-is-the-use-of-torch-no-grad-in-pytorch

Best wishes,
Xiaodong Cun

**Comment by mahmoudnafifi** [comment]
Hi Xiaodong, 

Thanks for pointing it out. I used it in the evaluation code used during training (not released yet), but I forgot it in the demos. I updated the code based on your recommendation. Thank you very much!

Best,
Mahmoud
